accelerator/config.c

```diff
@@ -517,3 +520,29 @@ void ta_core_destroy(ta_core_t* const core) {
   logger_destroy_json_serializer();
   br_logger_release();
 }
+
+void notify_initialization() {
```

Comment:
How about `notification_init` for consistency?

Reply:
sure

Reply:
But this function is sending a notification to bash script to inform the script that TA has been initialized.
Doesn't `notification_init`sound like initialized notification object?

Reply:
Or rename to `_trigger`

Reply:
For the sake of health check, such domain socket mechanism can be applied as well.